### ISSUE TITLE
Not possible to hard reset to current commit

### ISSUE THREAD

**Issue by mikebeaton** [ForkIssues/TrackerWin#2286]
Hi, I believe this issue applies on Mac and Windows. It is not possible to reset to the current commit: the "Reset '{branchname}' to Here..." menu item is hidden if right-clicking on the current commit. However, hard reset to the current commit is useful - in fact, something I do quite often - as a way of discarding all current non-committed changes.

**Comment by DanPristupov** [comment]
https://github.com/fork-dev/TrackerWin/issues/415#issuecomment-2014858881

**Comment by DanPristupov** [comment]
I'm not sure we should add reset for the active commit. If one wants to discard changes, they should do this explicitly (Ctrl+A, Ctrl+Shift+D). `reset` doesn't discard untracked files anyway...

**Comment by mikebeaton** [comment]
> `reset` doesn't discard untracked files anyway...

You're right, it doesn't. I use Fork as a convenient GUI for doing things which I know how to do in git anyway, and that is one command which I use a lot. (Perhaps rightly or wrongly, though I don't think it's totally unreasonable...! :-) )
